test/unit/org/apache/cassandra/cql3/validation/operations/TruncateTest.java

```diff
@@ -45,4 +53,12 @@ public void testTruncate() throws Throwable
             assertEmpty(execute("SELECT * FROM %s"));
         }
     }
+
+    @Test
+    public void testRemoteTruncateStmt() throws Throwable
+    {
+        System.setProperty("cassandra.remote_truncate_statement", RemoteTruncateStatement.class.getName());
+        createTable("CREATE TABLE %s (a int, b int, c int, PRIMARY KEY(a, b))");
+        execute("TRUNCATE TABLE %s");
```

Comment:
no assertions? how do you know it worked?

Reply:
@jacek-lewandowski Which assertion do you want to see here? Currently it succeeds if it doesn't fail with an exeption.

Reply:
I mean, how do you know it uses `RemoteTruncateStatement`?

Reply:
Good point. With obtaining the property string or the class from factory the test wasn't using it. Now I fixed the test, so it confirms that it utilises it.